cmd/openfga/openfga.go

```diff
@@ -56,6 +59,10 @@ type svcConfig struct {
 	// RequestTimeout is a limit on the time a request may take. If the value is 0, then there is no timeout.
 	RequestTimeout time.Duration `default:"0s" split_words:"true"`
 
+	TLSEnabled  bool   `default:"false" split_words:"true"`
+	TLSCertPath string `split_words:"true"`
```

Comment:
Now that I think about it..

We should have

```
GRPCTLSCertPath
GRPCTLSKeyPath

HTTPGatewayTLSCertPath
HTTPGatewayTLSKeyPath
```
because we probably don't want to enable TLS between the gateway proxy and the grpc server in our deployment of it, and they should be capable of being controlled individually.
----

Also, along with these additions, can I recommend that do make some changes to this Config structure like so?

```
type GRPCServerConfig struct {

    // Addr is the host:port address that the server will listen on.
    Addr string
    
    TLSCertPath string
    TLSKeyPath string
}

type HTTPGatewayConfig struct {
    // Addr is the host:port address that the HTTP gateway server will listen on.
    Addr string
    
    TLSCertPath string
    TLSCertKey string
```

type ServerConfiguration struct {
    ServiceName string
    GRPCServer GRPCServerConfig
    HTTPGateway HTTPGatewayConfig
    // ...etc
}